gamify-impl/src/main/java/ch/heigvd/gamify/domain/aggregate/RankingRepository.java

```diff
@@ -10,10 +10,11 @@
 public interface RankingRepository extends Repository<Rule, RuleIdentifier> {
   @Query(nativeQuery = true, value =
           "WITH subquery AS ( "
-          + "SELECT p.points AS total, p.idx_user_user_id AS userId, ROW_NUMBER() OVER (ORDER BY p.points DESC) as rank "
-          + "FROM end_user_points p "
-          + "WHERE p.idx_user_app_name = :app "
+          + "SELECT SUM(p.points) AS total, p.idx_end_user_user_id AS userId, ROW_NUMBER() OVER (ORDER BY SUM(p.points) DESC) as rank "
+          + "FROM end_user_point_award p "
+          + "WHERE p.idx_end_user_app_name = :app "
           + "AND p.idx_category_name = :category "
+          + "GROUP BY p.idx_end_user_user_id, p.points "
```

Comment:
Maybe without `p.points` aggregation.

```suggestion
          + "GROUP BY p.idx_end_user_user_id "
```